vscode/src/copilot/webview/copilot.css

```diff
@@ -0,0 +1,177 @@
+/* Copyright (c) Microsoft Corporation.
+   Licensed under the MIT License. */
+
+body {
+  padding: 12px;
+  height: 100%;
+  /*  match the native vscode chat style */
+  font-family:
+    Segoe WPC,
+    Segoe UI,
```

Comment:
I believe these font names should be in double quotes if they include spaces. (Or at least that's common practice even if it isn't required).